test/gtest/common/test_ucc.cc

```diff
@@ -247,6 +247,14 @@ UccJob::UccJob(int _n_procs, ucc_job_ctx_mode_t _ctx_mode, ucc_job_env_t vars) :
 {
     ucc_job_env_t env_bkp;
     char *var;
+
+    /* NCCL TL is disabled since it currently can not support non-blocking
+       team creation. */
+    vars.push_back({"UCC_TL_NCCL_TUNE", "0"});
+    /* GDR is temporarily disabled due to known issue that may result
+       in a hang in the destruction flow */
+    vars.push_back({"UCX_IB_GPU_DIRECT_RDMA", "no"});
```

Comment:
instead of disabling GDR here, is it ok to disable stop nv_peer_mem on test server?